hack/jenkins/jobs/bootkube_e2e.groovy

```diff
@@ -0,0 +1,61 @@
+// META
+
+job_dir = 'bootkube'
+
+// CONFIG
+
+//fork_to_use = 'bootkube'     // TODO: uncomment
+fork_to_use = 'chimera-mickens-io' // TODO: remove
+job_admins = ['sym3trid', 'dghubbled', 'kansd', 'chancezd', 'squatd', 'ericchiangd', 'ethernetdand', 'pbx0d', 'yifan-gud', 'aaronlevyd', 'derekparkerd', 'diegsd', 'hongchaodengd', 'xiang90d', 'abhinavdahiyad', 'rphillipsd', 'colemickens']
+user_whitelist = job_admins
+org_whitelist = ['coreos', 'coreos-inc']
+
+// JOBS
+network_providers = ['flannel', 'calico']
+network_providers.each { np ->
+  job_name = "bootkube-e2e-${np}"
+
+  pipelineJob(job_name) {
+    parameters {
+      stringParam('sha1', 'origin/master', 'git reference to build')
+    }
+    definition {
+      triggers {
+        githubPullRequest {
+          admins(job_admins)
+          userWhitelist(user_whitelist)
+          orgWhitelist(org_whitelist)
+          useGitHubHooks(true)
+          onlyTriggerPhrase(false)
+          triggerPhrase("coreosbot run ${job_name}")
```

Comment:
nit: make sure this can't conflict with the existing e2e jobs so we can run them in parallel

Reply:
The prefix in `context()` assures this. In general, that `context` is the identifier to GitHub for commit statuses. (aka, any client can clobber another, yes, etc.)

Reply:
sgtm, as long as it's documented here https://github.com/kubernetes-incubator/bootkube/blob/master/Documentation/development.md#running-pr-tests

Reply:
Ah. There's two things then. We don't want the build commit statuses to conflict, but we also want to trigger them independently.

In this case, I will be prefixing both the commit status, and the retrigger commands with "tku-", so that they are the same and easy to infer from one another, hopefully also alleviating the need to look them up as they'll be right there on the page.